### PR TITLE
CHK-1376-new-wallet-domain-app

### PR DESCRIPTION
**Pull request by Fernando-Granato**
### List of changes
Added new domain for wallet. 
For now the configuration provides only aks.

### Motivation and context
The change was made to allow the generation of a new domain app for the wallet project.

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/domains/wallet-app/99_main.tf [hunk 1]
```diff
@@ -0,0 +1,48 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "= 2.99.0"
```
Comment (infantesimone) [inline src/domains/wallet-app/99_main.tf:5]: better to start directly with terraform v3 (3.40.0)

Comment (infantesimone) [reply src/domains/wallet-app/99_main.tf:5]: you can refer to https://github.com/pagopa/io-infra/pull/422/files

Comment (Fernando-Granato) [reply src/domains/wallet-app/99_main.tf:5]: ok 🚀

**Comment by Fernando-Granato** [comment]
> can you split the PR? with only the `app` folder and then make a PR dedicated to `common`?

https://github.com/pagopa/pagopa-infra/pull/870 PR for common

**Review by pietro-tota — APPROVED** [review]
🚀

**Review by infantesimone — APPROVED** [review]

**Pull request merged**